app/webpacker/packs/global_styles.js

```diff
@@ -1,5 +1,5 @@
 import 'flag-icons/css/flag-icons.css';
-import '@cubing/icons';
+import '@cubing/icons/css';
```

Comment:
This is the forward-compatible CSS import path:

- https://icons.cubing.net/#npm-usage
- https://github.com/cubing/icons/blob/main/package.json#L21-L23

Reply:
urgh, this worked just fine in dev `webpack` doesn't seem to handle this properly. Lemme see if I can add a shim using a folder.

Reply:
I've tried adding a workaround in `@cubing/icons@v2.1.0-pre.3` but this is still broken.

Unfortunately, I no longer use or debug webpack and don't want to spend any time debugging webpack, so I'm not sure what to do. https://webpack.js.org/guides/package-exports/ suggest this should be working.

Reply:
Trying a rollback of this (which I would be very unhappy about, but #yolo).

```suggestion
import '@cubing/icons';
```